.detekt/config.yml

```diff
@@ -506,15 +506,15 @@ style:
     active: false
     includeLineWrapping: false
   ForbiddenComment:
-    active: true
-    values: ['TODO:', 'FIXME:', 'STOPSHIP:']
+    active: false
+    values: ['TODO', 'FIXME', 'STOPSHIP']
     allowedPatterns: ''
   ForbiddenImport:
     active: false
     imports: []
     forbiddenPatterns: ''
   ForbiddenMethodCall:
-    active: false
+    active: true
     methods: ['kotlin.io.println', 'kotlin.io.print']
```

Comment:
Do we have legit cases where we need to call `kotlin.io.print` in production code?